bfxapi/enums.py

```diff
@@ -0,0 +1,50 @@
+from enum import Enum
+
+class OrderType(str, Enum):
+    LIMIT = "LIMIT"
+    EXCHANGE_LIMIT = "EXCHANGE LIMIT"
+    MARKET = "MARKET"
+    EXCHANGE_MARKET = "EXCHANGE MARKET"
+    STOP = "STOP"
+    EXCHANGE_STOP = "EXCHANGE STOP"
+    STOP_LIMIT = "STOP LIMIT"
+    EXCHANGE_STOP_LIMIT = "EXCHANGE STOP LIMIT"
+    TRAILING_STOP = "TRAILING STOP"
+    EXCHANGE_TRAILING_STOP = "EXCHANGE TRAILING STOP"
+    FOK = "FOK"
+    EXCHANGE_FOK = "EXCHANGE FOK"
+    IOC = "IOC"
+    EXCHANGE_IOC = "EXCHANGE IOC"
+
+class FundingOfferType(str, Enum):
+    LIMIT = "LIMIT"
+    FRR_DELTA_FIX = "FRRDELTAFIX"
+    FRR_DELTA_VAR = "FRRDELTAVAR"
+
+class Flag(int, Enum):
+    HIDDEN = 64
+    CLOSE = 512
+    REDUCE_ONLY = 1024
+    POST_ONLY = 4096
+    OCO = 16384
+    NO_VAR_RATES = 524288
+
+class Error(int, Enum):
+    ERR_UNK = 10000
+    ERR_GENERIC = 10001
+    ERR_CONCURRENCY = 10008
+    ERR_PARAMS = 10020
+    ERR_CONF_FAIL = 10050
+    ERR_AUTH_FAIL = 10100
+    ERR_AUTH_PAYLOAD = 10111
+    ERR_AUTH_SIG = 10112
+    ERR_AUTH_HMAC = 10113
+    ERR_AUTH_NONCE = 10114
+    ERR_UNAUTH_FAIL = 10200
+    ERR_SUB_FAIL = 10300
+    ERR_SUB_MULTI = 10301
+    ERR_SUB_UNK = 10302
+    ERR_SUB_LIMIT = 10305
+    ERR_UNSUB_FAIL = 10400
+    ERR_UNSUB_NOT = 10401
+    ERR_READY = 11000
```

Comment:
one thing, please configure your IDE to append newline to end of each file as it's linux standard